shared_requirements.txt

```diff
@@ -119,7 +119,7 @@ msrest>=0.6.21
 msrestazure<2.0.0,>=0.4.32
 azure-mgmt-core<2.0.0,>=1.3.0
 requests>=2.18.4
-uamqp~=1.6.0
+uamqp>=1.6.3
```

Comment:
since EH still uses >=1.6.0, this will prob give you an error. I think you can revert this.